aa_forum/templates/aa_forum/view/forum/new-topic.html

```diff
@@ -51,7 +50,13 @@
 {% endblock %}
 
 {% block extra_javascript %}
+<<<<<<< HEAD
     <script type="application/javascript" src="{% static "ckeditor/ckeditor-init.js" %}"></script>
     <script type="application/javascript" src="{% static "ckeditor/ckeditor/ckeditor.js" %}"></script>
     <script type="application/javascript" src="{% versioned_static "aa_forum/javascript/aa-forum-ckeditor.min.js" %}"></script>
+=======
+    <script type="application/javascript" src="{% versioned_static 'ckeditor/ckeditor-init.js' %}"></script>
+    <script type="application/javascript" src="{% versioned_static 'ckeditor/ckeditor/ckeditor.js' %}"></script>
+    <script type="application/javascript" src="{% versioned_static 'aa_forum/javascript/aa-forum-ckeditor.min.js' %}"></script>
+>>>>>>> Refactor direct board links
```

Comment:
I see a merge conflict here, no idea why this has not been picked up by GitHub

Reply:
I did a rebase against the latest from development. Git was able to merge without any conflicts. Let's see if it works now.

Reply:
It's still there